### PR TITLE
Add presidio-image-redactor to pr pipeline

### PR DESCRIPTION
**Pull request by navalev**
This PR includes:
1. build and test the image redactor in the pr pipeline
2. image-redactor pipfile.lock update to lock numpy
3. temporary disable one of the integration tests - as it yields different results between tesseract versions

### CODE REVIEW

**Review by omri374 — APPROVED** [review]

**Review by SharonHart — APPROVED** [review]

File: .pipelines/templates/deploy-stage.yml [hunk 1]
```diff
@@ -1,130 +1,170 @@
...
-        inputs:
-          azureSubscription: ${{ parameters.AZURE_SUBSCRIPTION }}
-          action: Swap Slots
-          webAppName: $(AppServiceName)
-          sourceSlot: staging
-          resourceGroupName: ${{ parameters.RESOURCE_GROUP_NAME }}
\ No newline at end of file
+                    ANALYZER_STATUS_CODE=$(curl -s -o /dev/null -w "%{http_code}" --connect-timeout 1 --max-time 10 $ANALYZER_STAGING_URI/health)
+                    ANONYMIZER_STATUS_CODE=$(curl -s -o /dev/null -w "%{http_code}" --connect-timeout 1 --max-time 10 $ANONYMIZER_STAGING_URI/health)
+                    IMAGE_REDACROE_STATUS_CODE=$(curl -s -o /dev/null -w "%{http_code}" --connect-timeout 1 --max-time 10 $IMAGE_REDACROE_STAGING_URI/health)
```
Comment (SharonHart) [inline .pipelines/templates/deploy-stage.yml:125]: typo in IMAGE_REDACROE_STAGING_URI

File: .pipelines/templates/deploy-stage.yml [hunk 2]
```diff
@@ -1,130 +1,170 @@
...
+                        echo analyzer status code is $ANALYZER_STATUS_CODE
+                        echo anonymizer status code is $ANONYMIZER_STATUS_CODE
+                        echo image-redactor status code is $IMAGE_REDACROE_STATUS_CODE
+                    done
+          - template: ./e2e-tests.yml
+            parameters:
+                test_suite: 'integration'
+                analyzer_base_url: $(ANALYZER_STAGING_URI)
+                anonymizer_base_url: $(ANONYMIZER_STAGING_URI)
+                image_redactor_base_url: $(IMAGE_REDACROE_STAGING_URI)
```
Comment (SharonHart) [inline .pipelines/templates/deploy-stage.yml:144]: image_redactor_base_url  Needs to be added to e2e-tests.yml and e2e framework, can be done when adding e2e tests

File: .pipelines/templates/deploy-stage.yml [hunk 3]
```diff
@@ -1,130 +1,170 @@
...
+                    echo "##vso[task.setvariable variable=STAGING_URI;isOutput=true]$URI"
+          - task: AzureWebAppContainer@1
+            displayName: Deploy to staging slot
+            inputs:
+                azureSubscription: ${{ parameters.AZURE_SUBSCRIPTION }}
+                deployToSlotOrASE: true
+                resourceGroupName: ${{ parameters.RESOURCE_GROUP_NAME }}
+                appName: ${{ parameters.IMAGE_REDACTOR_APP_NAME }}
+                slotName: staging
+                imageName: ${{ parameters.REGISTRY_NAME }}presidio-image-redactor:${{ parameters.TAG }}
```
Comment (SharonHart) [inline .pipelines/templates/deploy-stage.yml:102]: Can be a matrix?

File: .pipelines/templates/deploy-stage.yml (hunk 2 above)
Comment (navalev) [reply .pipelines/templates/deploy-stage.yml:144]: I'm going to remove the ci pipeline change and do it in a seperate PR

Comment (SharonHart) [reply .pipelines/templates/deploy-stage.yml]: Just what I wanted to suggest, remember that yesterday it was good for merge

**Pull request merged**